src/utils/Utils.ts

```diff
@@ -1345,4 +1347,8 @@ export default class Utils {
   public static isPlateIDValid(plateID): boolean {
     return /^[A-Z0-9- ]*$/.test(plateID);
   }
+
+  public static getHostname(): string {
```

Comment:
That helper can also be used in the following files: 
+ LockingManager
+ LoggingStorage